app/forms/mobile_number_form.rb

```diff
@@ -33,6 +33,11 @@ def save
   private
 
   def send_sms_message
+    if Rails.env.development?
+      Rails.logger.info("SMS CODE: #{OneTimePassword::Generator.new.code}")
```

Comment:
I think this probably won't yield the same value as that sent in the Notify SMS message below. I'm also not sure how we're storing the OTP code to check it's correct on the next 2FA page?